<head>
  <!-- Put fonts and css links here -->

  <meta charset="utf-8">
  <meta content="width=device-width, initial-scale=1.0" name="viewport">

  <title>Resources</title>
  <meta content="" name="description">
  <meta content="" name="keywords">

  <!-- Favicons -->
  <link rel="icon" type="image/png" href="../../content/induceR-favicon-w.png">

  <!-- (Google) Fonts -->
  <!-- Main CSS-->
  <link href="../../styles/index.css" rel="stylesheet">

  <script src='../../script/index.js' type="module"></script>

</head>

<body>
  <!-- Navigation menu -->
  <div class="navbar">
    <img id="nav-logo" src="..\..\content\induceR-favicon-w.png" alt="induceR">
    <div class="dropdown">
      <button class="dropbtn">Documentation
        <i class="fa fa-caret-down"></i>
      </button>
      <div class="dropdown-content">
        <a href="protocols.html">Protocols</a>
        <a href="labnotebook.html">Lab Notebook</a>
        <a href="biosafety.html">Biosafety and Ethics</a>
        <a href="resources.html">Resources</a>
      </div>
    </div>

    <div class="dropdown">
      <button class="dropbtn">Project
        <i class="fa fa-caret-down"></i>
      </button>
      <div class="dropdown-content">
        <a href="../project/background.html">Background</a>
        <a href="../project/design.html">Brainstorming</a>
        <a href="../project/drylabs.html">Dry Labs</a>
        <a href="../project/wetlabs.html">Wet Labs</a>
      </div>
    </div>

    <div class="dropdown">
      <button class="dropbtn">About
        <i class="fa fa-caret-down"></i>
      </button>
      <div class="dropdown-content">
        <a href="../about/team.html">Our Team</a>
        <a href="../about/collab.html">Collaborators</a>
        <a href="../about/sponsors.html">Sponsors</a>
      </div>
    </div>

    <a href="../../index.html">Home</a>
  </div>

  <div class="main">
    <h1>Resources</h1>
    <div class="section1">
      <h2>Citations</h2>

      1. Li Z, Nair SK. Quorum sensing: How bacteria can coordinate activity and synchronize their response to external
      signals? Protein Science : A Publication of the Protein Society. [Online] Wiley-Blackwell; 2012;21(10): 1403.
      Available from: doi:10.1002/PRO.2132 [Accessed: 1st October 2021]<br><br>

      2. Ng W-L, Bassler BL. Bacterial Quorum-Sensing Network Architectures. Annual review of genetics. [Online] NIH
      Public Access; 2009;43: 197. Available from: doi:10.1146/ANNUREV-GENET-102108-134304 [Accessed: 1st October 2021]<br><br>

      3. Swofford CA, Dessel V, Forbes NS, Jain RK. Quorum-sensing Salmonella selectively trigger protein expression
      within tumors. Available from: doi:10.1073/pnas.1414558112 [Accessed: 1st October 2021]<br><br>

      4. Silva KPT, Chellamuthu P, Boedicker JQ. Quantifying the strength of quorum sensing crosstalk within microbial
      communities. PLoS Computational Biology. [Online] Public Library of Science; 2017;13(10). Available from:
      doi:10.1371/JOURNAL.PCBI.1005809 [Accessed: 1st October 2021]<br><br>

      5. Schauder S, Bassler BL. The languages of bacteria. Genes & Development. [Online] Cold Spring Harbor Laboratory
      Press; 2001;15(12): 1468–1480. Available from: doi:10.1101/GAD.899601 [Accessed: 1st October 2021]<br><br>

      6. Hudaiberdiev S, Choudhary KS, Vera Alvarez R, Gelencsér Z, Ligeti B, Lamba D, et al. Census of solo LuxR genes
      in prokaryotic genomes. Frontiers in Cellular and Infection Microbiology. [Online] Frontiers; 2015;0(MAR): 20.
      Available from: doi:10.3389/FCIMB.2015.00020<br><br>

      7. Shong J, Huang Y-M, Bystroff C, Collins CH. Directed Evolution of the Quorum-Sensing Regulator EsaR for
      Increased Signal Sensitivity. ACS Chemical Biology. [Online] American Chemical Society; 2013;8(4): 789–795.
      Available from: doi:10.1021/cb3006402<br><br>

      8. Jumper J, Evans R, Pritzel A, Green T, Figurnov M, Ronneberger O, et al. Highly accurate protein structure
      prediction with AlphaFold. Nature. [Online] 2021;596(7873): 583–589. Available from:
      doi:10.1038/s41586-021-03819-2<br><br>

      9. Kelley LA, Mezulis S, Yates CM, Wass MN, Sternberg MJE. The Phyre2 web portal for protein modeling, prediction
      and analysis. Nature protocols. [Online] 2015/05/07. 2015;10(6): 845–858. Available from:
      doi:10.1038/nprot.2015.053<br><br>

      10. Källberg M, Wang H, Wang S, Peng J, Wang Z, Lu H, et al. Template-based protein structure modeling using the
      RaptorX web server. Nature protocols. [Online] 2012;7(8): 1511–1522. Available from: doi:10.1038/nprot.2012.085<br><br>

      11. Kim DE, Chivian D, Baker D. Protein structure prediction and analysis using the Robetta server. Nucleic acids
      research. [Online] Oxford University Press; 2004;32(Web Server issue): W526–W531. Available from:
      doi:10.1093/nar/gkh468<br><br>

      12. Wang Y, Virtanen J, Xue Z, Zhang Y. I-TASSER-MR: automated molecular replacement for distant-homology proteins
      using iterative fragment assembly and progressive sequence truncation. Nucleic acids research. [Online] Oxford
      University Press; 2017;45(W1): W429–W434. Available from: doi:10.1093/nar/gkx349<br><br>

      13. Grosdidier A, Zoete V, Michielin O. SwissDock, a protein-small molecule docking web service based on EADock
      DSS. Nucleic Acids Research. [Online] Oxford University Press; 2011;39(Web Server issue): W270. Available from:
      doi:10.1093/NAR/GKR366 [Accessed: 30th September 2021]<br><br>

      14. Volkamer A, Kuhn D, Rippmann F, Rarey M. DoGSiteScorer: a web server for automatic binding site prediction,
      analysis and druggability assessment. Bioinformatics. [Online] Oxford Academic; 2012;28(15): 2074–2075. Available
      from: doi:10.1093/BIOINFORMATICS/BTS310 [Accessed: 30th September 2021]<br><br>

      15. Laskowski RA, Swindells MB. LigPlot+: Multiple Ligand–Protein Interaction Diagrams for Drug Discovery. Journal
      of Chemical Information and Modeling. [Online] American Chemical Society; 2011;51(10): 2778–2786. Available from:
      doi:10.1021/CI200227U [Accessed: 30th September 2021]<br><br>

      16. Pruitt KD, Tatusova T, Maglott DR. NCBI reference sequences (RefSeq): a curated non-redundant sequence
      database of genomes, transcripts and proteins. Nucleic Acids Research. [Online] Oxford University Press;
      2007;35(Database issue): D61. Available from: doi:10.1093/NAR/GKL842 [Accessed: 30th September 2021]<br><br>

      17. New England BioLabs. Tm Calculator. [Online] Available from: https://tmcalculator.neb.com/#!/main [Accessed:
      30th August 2021]<br><br>

      18. Isalan M, Choo Y. Phage display of zinc fingers and other nucleic acid - binding motifs. In: Clackson Lowman,
      Henry B. T (ed.) Phage display : a practical approach. [Online] Oxford; New York: Oxford University Press; 2004.
      Available from: http://site.ebrary.com/id/10266414<br><br>

      19. Collins CH, Arnold FH, Leadbetter JR. Directed evolution of Vibrio fischeri LuxR for increased sensitivity to
      a broad spectrum of acyl-homoserine lactones. Molecular microbiology. [Online] England; 2005;55(3): 712–723.
      Available from: doi:10.1111/j.1365-2958.2004.04437.x<br><br>
    </div>

    <div class="section2">
      <h2>Data & Computing Codes</h2>
    </div>

    <div class="section1">
      <h2>Data Accessibility Statements</h2>
      <p>The Imperial iDEC team declares that all data can be accessed on the links provided above. We did not violate
        any GDPR. </p>
    </div>

  </div>
</body>

<a href="#">
  <img id="back-to-top" src="../../content/back-to-top.svg">
</a>

<footer>Imperial College London iDEC 2021 Team ©</footer>