<head>
  <!-- Put fonts and css links here -->

  <meta charset="utf-8">
  <meta content="width=device-width, initial-scale=1.0" name="viewport">

  <title>Dry Labs</title>
  <meta content="" name="description">
  <meta content="" name="keywords">

  <!-- Favicons -->
  <link rel="icon" type="image/png" href="../../content/induceR-favicon-w.png">

  <!-- (Google) Fonts -->
  <!-- Main CSS-->
  <link href="../../styles/index.css" rel="stylesheet">

  <script src='../../script/index.js' type="module"></script>

</head>

<body>
  <!-- Navigation menu -->
  <div class="navbar">
    <img id="nav-logo" src="..\..\content\induceR-favicon-w.png" alt="induceR">
    <div class="dropdown">
      <button class="dropbtn">Documentation
        <i class="fa fa-caret-down"></i>
      </button>
      <div class="dropdown-content">
        <a href="../documentation/protocols.html">Protocols</a>
        <a href="../documentation/labnotebook.html">Lab Notebook</a>
        <a href="../documentation/biosafety.html">Biosafety and Ethics</a>
        <a href="../documentation/resources.html">Resources</a>
      </div>
    </div>

    <div class="dropdown">
      <button class="dropbtn">Project
        <i class="fa fa-caret-down"></i>
      </button>
      <div class="dropdown-content">
        <a href="background.html">Background</a>
        <a href="design.html">Design</a>
        <a href="drylabs.html">Dry Labs</a>
        <a href="wetlabs.html">Wet Labs</a>
      </div>
    </div>

    <div class="dropdown">
      <button class="dropbtn">About
        <i class="fa fa-caret-down"></i>
      </button>
      <div class="dropdown-content">
        <a href="../about/team.html">Our Team</a>
        <a href="../about/collab.html">Collaborators</a>
        <a href="../about/sponsors.html">Sponsors</a>
      </div>
    </div>

    <a href="../../index.html">Home</a>

  </div>

  <div class="banner">
    <img id="codons" src="../../content/banners/codons.svg">
    <h1 class="banner-head">Dry Labs</h1>
  </div>

  <img id="el-membrane" src="..\..\content\el_membrane.svg" alt="membrane">

  <div class="main">

    <div class="section1">
      <div class="main-container">
        <h2 id="stage-title0">3D visualisation instructions</h2>
        <div id="mol-instructions">
          Use mouse to manipulate the 3D render. <br><br>
          To rotate: drag molecule to desired direction.<br><br>
          To zoom: scroll or hold shift + drag downwards to zoom in/upwards to zoom out.<br><br>
          To pan: hold ctrl + drag to desired direction.
        </div>

        <div class='mol_viewer'>
          <script src="https://3Dmol.csb.pitt.edu/build/3Dmol-min.js"></script>
          <div class='viewer_3Dmoljs' data-href='..\..\content\molecules\All Predicted WT EsaR.pdb'
            data-backgroundcolor='white' data-style='cartoon:color=spectrum'>
            1H0M
          </div>
          <h4>3D structure of all E<sub>saR</sub> protein structure predictions</h4>
        </div>
      </div>

    </div>

    <img id="el-membrane" src="..\..\content\el_membrane.svg" alt="membrane">

    <div class="section2">
      <div class="main-container">
        <h3 id="stage-title1">STRUCTURE PREDICTION</h3>
        <div id="stage1">
          Structure prediction of E<sub>saR</sub> was conducted using
          <a href="https://www.nature.com/articles/nprot.2012.085">RaptorX</a>,
          <a href="https://currentprotocols.onlinelibrary.wiley.com/doi/10.1002/0471250953.bi0508s52">I-TASSER</a>,
          <a href="https://www.nature.com/articles/nprot.2009.2">Phyre</a>, and
          <a href="https://www.nature.com/articles/s41586-021-03819-2_reference.pdf">Alphafold</a>.
          For I-TASSER, the EsaR homologue TraR was submitted as a template (PDB: 1H0M). After prediction, the obtained
          conformations were aligned in PyMOL.
        </div>

        <img id="stagefig1" src="../../content/drylab/fig1.jpg">

        <h3 id="stage-title2">LIGAND DOCKING</h3>
        <div id="stage2">
          <a href="https://academic.oup.com/nar/article/39/suppl_2/W270/2506492">SwissDock</a>
          server (Swiss Institute of Bioinformatics) was used to perform molecular docking of the cognate 3OC6HSL. We
          restricted docking to a docking box enclosing the binding pocket which was identified by the cavity-detection
          method provided by the DoGSiteScorer tool from the ProteinsPlus server.<br><br>
          The <a href="https://doi.org/10.1093/protein/8.2.127">LigPlot</a> tool was used to identify the interactions
          between the ligand and the E<sub>saR</sub> predicted conformations, which were then compared to that of the
          ligand and protein crystal structures homologous to E<sub>saR</sub>. These suitable homologous structures were
          selected by running a BLAST search and filtering the results bound to their respective cognate ligand
          available from the Protein Data Bank.
        </div>

        <img id="stagefig2" src="../../content/drylab/lig.png">

        <img id="stagefig3" src="../../content/drylab/insilico.png">

        <h3 id="stage-title3">MUTANT ANALYSIS</h3>
        <div id="stage3">
          Python scripts were written to substitute the selected 4 residues with the 12 mutagenic NDT codons, generating
          a total of 20,736 (12<sup>4</sup>) mutants in a combinatorial process. Scripts were also written to perform
          single-residue mutagenesis, substituting each of the 4 residues with the 20 amino acids and generating 80 (20
          x 4) single-mutation sequences. DNA and residue mutant sequences were generated as input to downstream
          prediction software.<br><br>
          AlphaFold codes were modified to read sequences stored in a .csv file instead of importing one sequence at a
          time, hence automating and accelerating the prediction. The structures of the first 50 mutants from the
          20,736-mutant library were predicted to study the effect of the mutation on binding sites.<br><br>

        </div>
      </div>
    </div>

  </div>
</body>

<a href="#">
  <img id="back-to-top" src="../../content/back-to-top.svg">
</a>

<footer>Imperial College London iDEC 2021 Team Â©</footer>